src/app/navbar/navbar.component.ts

```diff
@@ -69,6 +71,7 @@ export class NavbarComponent extends Logout implements OnInit {
       }
     });
     this.userQuery.extendedUserData$.pipe(takeUntil(this.ngUnsubscribe)).subscribe((extendedUser) => (this.extendedUser = extendedUser));
+    this.navbarService.activePage$.pipe(takeUntil(this.ngUnsubscribe)).subscribe((activePage) => (this.activePage = activePage));
```

Comment:
shouldn't need to manually set and activePage. The activePage can probably be figured out using some field within activatedRoute. If activatedRoute isn't possible. this should not be a subscribe. an async pipe in the html should do

Reply:
Removed that in my latest PR. Setting active state for the navbar and menu items using `routerLinkActive` in the html now